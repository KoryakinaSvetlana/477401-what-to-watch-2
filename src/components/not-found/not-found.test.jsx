import React from "react";
import renderer from "react-test-renderer";
import {BrowserRouter as Router} from "react-router-dom";
import NotFound from "./not-found";

it(`NotFound correctly render`, () => {
  const tree = renderer.create(
      <Router><NotFound /></Router>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
