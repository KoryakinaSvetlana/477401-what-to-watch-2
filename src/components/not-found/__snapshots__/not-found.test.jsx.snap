// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound correctly render 1`] = `
<div>
  <span>
    Страница не найдена. 
  </span>
  <a
    href="/"
    onClick={[Function]}
  >
    Вернуться на главную
  </a>
</div>
`;
